Formula/acl2.rb

```diff
@@ -0,0 +1,36 @@
+class Acl2 < Formula
+  desc "The logic and programming language in which you can model computer systems"
+  homepage "https://www.cs.utexas.edu/users/moore/acl2/index.html"
+  url "https://github.com/acl2/acl2/archive/8.3.tar.gz"
+  sha256 "45eedddb36b2eff889f0dba2b96fc7a9b1cf23992fcfdf909bc179f116f2c5ea"
+  license "BSD-3-Clause"
+
+  depends_on "sbcl"
+
+  def install
+    prefix.install Dir["*"]
+    system "make",
+           "-C", prefix,
+           "-j", ENV.make_jobs,
+           "LISP=#{HOMEBREW_PREFIX}/bin/sbcl",
+           "ACL2=#{prefix}/saved_acl2",
+           "USE_QUICKLISP=0",
+           "all", "basic"
+    ln_s prefix/"saved_acl2", bin/"acl2"
+    system "make",
```

Comment:
Why is this run twice?

Reply:
Because ACL2 consists several applications `acl2` and `acl2p`.
- `acl2` is the original theorem prover
- `acl2p` is the theorem prover, which supports parallel theorem proving.
The second operation build `acl2p` with the `ACL2_PAR` option.